tracker/emitter.go

```diff
@@ -90,7 +90,7 @@ func InitEmitter(options ...func(*Emitter)) *Emitter {
 
 	// Setup default event storage
 	if e.Storage == nil {
-		e.Storage = *InitStorageSQLite3(e.DbName)
+		e.Storage = *InitStorageMemory()
```

Comment:
Do other teams use this library at Thalmic too? Would this affect any other team in case of they're using the default value?

Reply:
Changing the default is unfortunately the only way we are going to be able to build this tracker without sqlite. Luckily, no one else is using this library yet (I forked it from snowplow's repo for the sake of making this change).